app/controllers/equipment_models_controller.rb

```diff
@@ -31,7 +31,7 @@ def index
     end
   end
 
-  def show
+  def show # rubocop:disable AbcSize, MethodLength
```

Comment:
We might still be failing the `AbcSize` metric since we're adding more assignments, but that's fine for now.